pkg/admissioncontroller/webhook/admission/resourcesize/handler.go

```diff
@@ -100,6 +105,17 @@ func (h *Handler) handle(req admission.Request) error {
 	return nil
 }
 
+func relevantObjectSize(rawObject []byte) (int64, error) {
+	var obj map[string]any
+	err := json.Unmarshal(rawObject, &obj)
+	if err != nil {
+		return 0, err
+	}
+	delete(obj, "status")
+	marshalled, err := json.Marshal(obj)
```

Comment:
Missing error check on marshalling
```suggestion
	marshalled, err := json.Marshal(obj)
	if err != nil {
		return 0, err
	}	
```